app/views/register/asset/shares/ShareCompanyNameView.scala.html

```diff
@@ -38,7 +38,8 @@
             field = form("value"),
             label = messages("shareCompanyName.heading"),
             labelClass=Some("heading-large"),
-            labelAsHeading = true
+            labelAsHeading = true,
+            autocomplete = Some("name")
         )
```

Comment:
@formHelper(action = ShareCompanyNameController.onSubmit(mode, index, draftId), 'autoComplete -> "off")
..... should 'autoComplete->"on" ?